app/services/irs_attempts_api/tracker_events.rb

```diff
@@ -13,6 +13,15 @@ def account_reset_account_deleted(success:, failure_reason: nil)
       )
     end
 
+    # @param [String] determination One of 'rejected' or 'activated'
+    # A profile offlined for review has been approved or rejected.
+    def fraud_review_adjudicated(determination:)
+      track_event(
+        :fraud_review_adjudicated,
+        determination: determination,
+      )
+    end
```

Comment:
We keep the methods alphabetized and I just plunked this in the top. We probably want to bikeshed this name a little. Is the term "adjudicated" actually used?

Reply:
And, related: Do we think it would be better if "the account was reviewed and deemed fraudulent" and "the account was reviewed and deemed non-fraudulent" were two separate events, or the same event (as I have here) with a `determination` parameter? It is presently trivial to change this.

Reply:
I would consider calling it `decision` instead of `determination` but otherwise I like the method name